nautobot/core/filters.py

```diff
@@ -439,6 +439,11 @@ class NaturalKeyOrPKMultipleChoiceFilter(django_filters.ModelMultipleChoiceFilte
     field_class = forms.MultiMatchModelMultipleChoiceField
 
     def __init__(self, *args, prefers_pk=False, **kwargs):
+        """Initialize the NaturalKeyOrPKMultipleChoiceFilter.
+
+        Args:
+            prefers_pk (bool, optional): Prefer PK (ID) over the 'to_field_name'. Defaults to False.
```

Comment:
Wonder if we should rename the arg to `prefers_id` since that's what it actually uses?

Reply:
I was wondering the same, time for find/replace.